_src/_assets/css/_button.scss

```diff
@@ -0,0 +1,24 @@
+.btn {
+  display: inline-block;
+  padding: .75em 1.25em;
+  font-size: 1em;
+  font-weight: 500;
+  color: #fff;
+  text-align: center;
+  text-decoration: none;
+  vertical-align: middle;
+  cursor: pointer;
+  user-select: none;
+  background-color: $link-color;
+  border: 1px solid $link-color;
+  border-radius: .5em;
+
+  &:hover {
+    border-color: darken($link-color, 5%);
```

Comment:
I think this is evaluating out to a blue-on-blue treatment on `:hover`; could we use white (or similar) here? (And if we could include a `:focus` style here as well, 😍😍😍)

Reply:
Correct, I added a more styles to `<a>` and forgot to double check this one. Updated!